shell/browser/browser_mac.mm

```diff
@@ -393,4 +394,26 @@ void RemoveFromLoginItems() {
   return true;
 }
 
+bool Browser::IsSecureInputEnabled() {
+  // See
+  // https://developer.apple.com/library/content/technotes/tn2150/_index.html
+  return IsSecureEventInputEnabled();
+}
+
+void Browser::SetSecureInputEnabled(bool enabled) {
+  // See
+  // https://developer.apple.com/library/content/technotes/tn2150/_index.html
+  if (enabled) {
```

Comment:
calling `setSecureInputEnabled()` with the same value twice should have no effect IMO.